model/src/main/java/com/epam/esm/model/parameters/CertificateParameter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.model.parameters;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum CertificateParameter {
+    TAG_NAME("tagName", TableColumnName.TAG_DAO_NAME),
+    CERTIFICATE_NAME("name", TableColumnName.CERTIFICATE_DAO_NAME),
+    CERTIFICATE_DESCRIPTION("description", TableColumnName.CERTIFICATE_DAO_DESCRIPTION),
+    CREATE_DATE("createDate", TableColumnName.CERTIFICATE_DAO_CREATE_DATE);
+
+    String name;
```

Comment:
is there any reason why the following two fields have package access level?

Reply:
Will be fixed in the next PR